tests/utils/exec/rosacli/cluster_service.go

```diff
@@ -139,6 +140,10 @@ func (c *clusterService) ReflectClusterDescription(result bytes.Buffer) (res *Cl
 	return res, err
 }
 
+func (c *clusterService) RetrieveHelpForClusterCreate() (output bytes.Buffer, err error) {
```

Comment:
Just name it `Help`

Reply:
Why not reusing `Create()` with the `-h` flag instead of creating a new method ?